### PR TITLE
Popup Menu color change fix (Mix mode)

### PR DESCRIPTION
**Pull request by ZimCodes**
Fixes the popup menu only changing color on _New Tab_ pages in mix mode.
#### Description

If the user is on a non _New Tab_ page and decides to create a _New Tab_. On the _New Tab_ page, the popup menu will change colors to match the current theme. However if the user goes back to the non _New Tab_ page, the popup menu will not change to match the current theme for that tab. It will instead switch back to the default color of Doki Theme. 

Also, all non _New Tab_ pages now have a random theme applied to them.

##### Why?
This is because the old tab is not being recorded as part of the mixed tab collection.

#### Motivation and Context

Popup menu can now change colors on pages other than _New Tab_ pages.
#### How Has This Been Tested?

This has been tested using mozilla's _web-ext_ command line tool & the browser's built-in extension inspector
#### Screenshots (if appropriate):
Problem:
https://user-images.githubusercontent.com/23222943/103421114-b0e46a80-4b92-11eb-882b-da77d459b5f7.mp4
Solution:
https://user-images.githubusercontent.com/23222943/103421126-bd68c300-4b92-11eb-9756-3d875bc1ed61.mp4
#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-Functional Change (non-user facing changes)

#### Checklist:

- [x] I updated the version (base package.json and re-built the master extension).
- [x] I updated the changelog with the new functionality.

### CODE REVIEW

**Comment by ZimCodes** [comment]
And it all started 20 days ago...

**Review by Unthrottled — COMMENTED** [review]
Works as advertised!

Any reason why you got rid of the `.editorconfig` file that I put in?

**Comment by ZimCodes** [comment]
> Any reason why you got rid of the `.editorconfig` file that I put in?

Did not know it was needed.

In any case, I would like to push one more thing to this.

**Comment by ZimCodes** [comment]
@Unthrottled Alright its ready.

**Comment by ZimCodes** [comment]
@Unthrottled Now... This branch is ready to be merged.

File: firefoxThemes/popup/popup.css [hunk 1]
```diff
@@ -20,10 +25,19 @@ h1 {
...
+        top:2.5em;
+    }
+}
 img {
-    display: block;
+    position: relative;
     margin: 1em auto;
-    max-width: 40%;
+    max-width: 30%;
+    animation: bounce 1.2s ease-in-out infinite alternate;
```
Comment (Unthrottled) [inline firefoxThemes/popup/popup.css:40]: This caught me off guard at first! It is an interesting addition though :+1:

**Review by Unthrottled — APPROVED** [review]
Those where some interesting last minute updates. I've got one more thing that I want to sneak in though

**Comment by Unthrottled** [comment]
> Did not know it was needed.

Yup. I prefer the compactness of 2 spaces, this should make [our editors](https://editorconfig.org/) get on the same page. If the loss it was an artifact of your merge, do you want some help with cleaning up your git history? 

If it was intentional, please refrain from deleting existing things without a good reason. I will remain open-minded, if it communicated.

Don't be mistaken, I appreciate your work though!

File: firefoxThemes/popup/popup.js [hunk 2]
```diff
@@ -10,11 +10,12 @@ function setCss(chosenTheme) {
     const {colors} = chosenTheme.definition
     const styles = `
 .popup-header {
-background-color: ${colors.headerColor};
+  background-color: ${colors.headerColor};
+  color: ${colors.infoForeground};
```
Comment (Unthrottled) [inline firefoxThemes/popup/popup.js:14]: | before | after |
| --- | --- |
| image | image |

**Pull request merged**